src/components/VoiceRecorder/VoiceRecorder.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { addRecord } from './VoiceRecorder.actions';
+import { startRecord } from './VoiceRecorder.actions';
+import Mic from '@material-ui/icons/Mic';
```

Comment:
by convention we import 3rd party packages like `react` or `material-ui` first, then local files we're importing, so it's easier for other developers to know what comes from the outside and what from within the app.